### ISSUE TITLE
Identify and solve "table elements" in Cache listings

### ISSUE THREAD

**Issue by eddiemuc** [cgeo/cgeo#12910]
### Tell us your idea!

In some caches the cache description ends with the following text:

Diese Beschreibung enthält Tabellenelemente, die evtl. nur auf geocaching.com korrekt angezeigt werden.

Example: https://coord.info/GC3DGN4

I assume this refers to table markup usable in GC Listings which is not interpreted by c:geo?

Wouldn't it be possible to interpret this markup and thus render the cache listing correctly?

### What solution would you suggest?

Interpret GC.com table markup and replace b corresponding HTML code

### Is there an alternative?

Leave everything as is

**Comment by fm-sys** [comment]
Yes, that would be great.

Please note that tables can contain images as well. Example: https://coord.info/GC1F03A

**Comment by Lineflyer** [comment]
> I assume this refers to table markup usable in GC Listings which is not interpreted by c:geo?

Correct. AFAIR it was not possible to draw this properly and therefore this warning message was introduced.

> Wouldn't it be possible to interpret this markup and thus render the cache listing correctly?

Either that or maybe its easier and more meaningful (for other use cases like colorings, etc.) to include some embedded "raw html view" which just shows the listing html part in a browser like fashion). That could be more multi purpose and probably easier?

**Comment by fm-sys** [comment]
There is a Library called [HtmlSpanner](https://github.com/NightWhistler/HtmlSpanner) which supports rendering of tables inside a textView... Can we use this library?

**Comment by eddiemuc** [comment]
> There is a Library called [HtmlSpanner](https://github.com/NightWhistler/HtmlSpanner) which supports rendering of tables inside a textView... Can we use this library? 

This issue is not about rendering html table elements like `<table>`, `<tr>`, `<td>`. It is about rendering the mockdown-like table syntax of GC.com (e.g. using `|`) as html tables. Does the lib support this?

**Comment by fm-sys** [comment]
Sneak preview:

||||
-|-|-
![grafik](https://user-images.githubusercontent.com/64581222/168677836-94df1a89-1b18-4b89-ae43-67731b34e20f.png)|![grafik](https://user-images.githubusercontent.com/64581222/168686228-96925d35-5b00-4c4d-9cad-9776a3d2d7f4.png)|![grafik](https://user-images.githubusercontent.com/64581222/168678169-82adfcc3-c663-419d-8d62-b70266b22522.png)

**Comment by eddiemuc** [comment]
While trying to work on this again, I found that lib com.osbcp:cssparser:1.7 is no longer available in mvnCentral (only in jcenter()). This lib is needed by HtmlSpanner. Shame.
